ingenialink/servo.py

```diff
@@ -997,9 +998,12 @@ def disturbance_write_data(self, channels, dtypes, data_arr):
             data_arr (list or list of list): Data array.
 
         """
-        data, chunks = self._disturbance_create_data_chunks(
-            channels, dtypes, data_arr, self.MAX_WRITE_SIZE
-        )
+        try:
+            data, chunks = self._disturbance_create_data_chunks(
+                channels, dtypes, data_arr, self.MAX_WRITE_SIZE
+            )
+        except OverflowError as e:
+            raise ILValueError(f"Disturbance data cannot be written. Reason: {e}")
```

Comment:
You can use `raise ILValueError(f"Disturbance data cannot be written.") from e`

Reply:
Done